src/operations/create_collection.ts

```diff
@@ -167,7 +167,14 @@ export class CreateCollectionOperation extends CommandOperation<Collection> {
 
     if (encryptedFields) {
       // Create the required index for queryable encryption support.
-      const createIndexOp = new CreateIndexOperation(db, name, { __safeContent__: 1 }, {});
+      // We could use `this.db.collection(name).createIndex()` to create the index,
```

Comment:
I think you can remove the "we could" because now that we have this improvement (nice work!) we wouldn't want to anymore

Reply:
Thanks! 

Either way - but fwiw I think eventually we might want to run all this in a transaction, in which case we probably would want to use `db.collection(..).createIndex()`.  

Using public APIs feels cleaner, and has the added benefit of a bit of extra testing cause we're using a complete public API, but I don't feel strongly.

Reply:
As said in the other thread I hesitate to adopt public API because of the extra cost it adds to work that can be done with less. But I don't think there's a universal rule. 

Anyway something like what you suggest sounds like it would be a drivers wide change, want to file a ticket about it? I'd prefer the comment be paired down, suggestions like this don't have a enough context to clue the next dev into what the intention was here and it can imply that the change is small and simple when it may have larger impacts (like re-selecting a server)